ccd-definition/CaseField.json

```diff
@@ -163,5 +163,38 @@
     "HintText": "This is optional - the claimant will see your answer.",
     "FieldType": "TextArea",
     "SecurityClassification": "Public"
+  },
+  {
+    "LiveFrom": "01/01/2017",
+    "CaseTypeID": "UNSPECIFIED_CLAIMS",
+    "ID": "extensionAccepted",
+    "Label": "Do you agree to extend the deadline to this date?",
+    "FieldType": "YesOrNo",
+    "SecurityClassification": "Public"
+  },
+  {
+    "LiveFrom": "01/01/2017",
+    "CaseTypeID": "UNSPECIFIED_CLAIMS",
+    "ID": "extensionCounter",
+    "Label": "Would you like to set a different extended deadline?",
+    "FieldType": "YesOrNo",
+    "SecurityClassification": "Public"
+  },
+  {
+    "LiveFrom": "01/01/2017",
+    "CaseTypeID": "UNSPECIFIED_CLAIMS",
+    "ID": "extensionCounterDate",
+    "Label": "New deadline",
+    "HintText": "This can't be later than 28 working days after the current deadline. All deadlines require a response before 4pm on the deadline day.",
+    "FieldType": "Date",
+    "SecurityClassification": "Public"
+  },
+  {
+    "LiveFrom": "01/01/2017",
+    "CaseTypeID": "UNSPECIFIED_CLAIMS",
+    "ID": "extensionResponse",
+    "Label": "Add a response",
+    "FieldType": "TextArea",
+    "SecurityClassification": "Public"
```

Comment:
@swalker125 - as per CMC-363, can you rename these to start with 'defendantSolicitor1claimResponseExtension'{attribute}, please? 

This is to make sure we know what thing the extension response is related to and that it's future proofed for multiparty defence solicitors.

Also documenting this is assuming that there is only one response per extension request from either claimant solicitor. So, if there were two, it would be envisaged that the UI is available to both and they need to agree offline. 

Business response -"Yes, i think we would just have the onescreen with the proposed extension date and either sol could update"

Reply:
have updated the naming as suggested